src/js/services/pull-request-storage-accessor.ts

```diff
@@ -30,6 +31,36 @@ const PullRequestStorageAccessor = () => {
     return data[key] ? JSON.parse(data[key]) : [];
   };
 
+  const syncIgnoredPrs = async (record: PullRequestRecord): Promise<Partial<PullRequest>[]> => {
+    const ignoredPrsString = await SettingsStorageAccessor().loadIgnoredPrs();
+    if (!ignoredPrsString.length) {
+      return [];
+    }
+
+    const ignoredPrs = ignoredPrsString.split(',').map(parsePullRequest);
+    const allPrs = Object.values(record).flat();
+
+    return ignoredPrs.filter(pr => {
+      const isPresent = allPrs.some(
+          recordPr => recordPr.ownerAndName === pr.ownerAndName && recordPr.number === pr.number,
+      );
```

Comment:
Business logic of PR is duplicated here. I found this confusing at https://github.com/renuo/github-pull-request-counter/pull/48/files#diff-cc88fa21c49af24149f9f2130ec6b144fd749624594803da9d6689c8efcaa084R61

Reply:
true, will refactor this